distributed/tests/test_active_memory_manager.py

```diff
@@ -1082,41 +1070,41 @@ async def tensordot_stress(c):
     da = pytest.importorskip("dask.array")
 
     rng = da.random.RandomState(0)
-    a = rng.random((20, 20), chunks=(1, 1))
-    b = (a @ a.T).sum().round(3)
-    assert await c.compute(b) == 2134.398
+    a = rng.random((10, 10), chunks=(1, 1))
+    # dask.array.core.PerformanceWarning: Increasing number of chunks by factor of 10
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        b = (a @ a.T).sum().round(3)
+    assert await c.compute(b) == 245.394
```

Comment:
Interestingly, (20, 20) would take forever or hang on my PC with `Worker=Worker`, but took as little as 5s with `Worker=Nanny`. On CI, `Worker=Nanny` hangs because of #5371.